Sources/RepositoryListRow.swift

```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+
+enum Destination: Identifiable {
+    var id: Int {
+        switch self {
+        case let .details(repository):
+            return repository.id
+        case let .user(owner):
+            return owner.id
+        }
+    }
+    
+    case details(repository: Repository), user(owner: Owner)
+}
```

Comment:
Move this to separate file.
You can reorganize your project structure like sth similar to this:


- Sources
  - Managers
    - ApiManager.swift
    - ...
  - Screens
    - RepositoryList
      - RepositoryListView.swift
      - RepositoryListViewModel.swift
    - RepositoryDetail
      - RepositoryDetailView.swift
      - RepositoryDetailViewModel.swift
      - ...
  - Components
    - RepositoryListRow.swift
    - ...
  - Models
    - Repository.swift
    - Owner.swift
    - Destination.swift
    - ...

etc. you get the idea 🤓